src/query/encoding.ts

```diff
@@ -21,13 +21,34 @@ export interface EncodingQuery {
   bin?: boolean | BinQuery | ShortEnumSpec;
   scale?: boolean | ScaleQuery | ShortEnumSpec;
 
+  sort?: SortOrder | SortField;
+
   field?: Field | EnumSpec<Field> | ShortEnumSpec;
   type?: Type | EnumSpec<Type> | ShortEnumSpec;
   // TODO: value
 
   // TODO: axisQuery, legendQuery
 }
 
+export enum SortOrder {
```

Comment:
These are not different from Vega-Lite at all, why don't you just import from Vega-Lite? 
(My previous comment already suggests this.)